hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -815,6 +841,91 @@
       stage: V2
       tests: []
     }
+    {
+      name: aon_wake_resume
+      desc: '''
+            Verify that the AON/Wake module is able to detect Resume Signaling and indicate that
+            via the USB device register interface.
+
+            - Connect the USB device to the USB host model.
+            - Send Suspend Signaling to the USB device.
+            - Verify that the 'host_lost' interrupt is raised indicating the lack of SOF signaling
```

Comment:
Wouldn't this be the `link_suspend` interrupt, not `host_lost`? Am I miscalculating? `host_lost` was supposed to take 4 frames, while we suspend in 3, I thought (which brings the link inactive, zeroing the missed SOF count).

Reply:
In other words, `host_lost` only triggers when the link is actually active / switching, but we never find SOF packets.

Reply:
The phrase about the associated cause also now doesn't match, since it still describes the `host_lost` interrupt. :)

Thanks for the alignments!